internal/operator-controller/applier/watchnamespace_test.go

```diff
@@ -71,59 +71,54 @@ func TestGetWatchNamespace(t *testing.T) {
 			},
 			expectError: false,
 		}, {
-			name: "cluster extension configures a watch namespace through annotation",
+			name: "cluster extension configures a watch namespace through spec.config.inline",
```

Comment:
do we already have a unit test under this banner? should we just remove this one?

Reply:
Yup, looks like we do. Removed this.